### ISSUE TITLE
AFter updating to Electron 27, JAWS screen reader does not read curly braces ({}) in the code editor

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#36239]
Type: <b>Bug</b>

AFter updating to Electron 27, JAWS screen reader does not read curly braces ({}) in the code editor evenn with the accessibility support enabled.
I tested from source code, and it starts failing after checking out the commit 12fece34ed49c972d85b1a9ff0f5b8532eea569c

VS Code version: Code - OSS Dev 1.85.0 (Commit 12fece34ed49c972d85b1a9ff0f5b8532eea569c, Date unknown)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-1185G7 - 3.00GHz (8 x 2995)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.85GB (6.69GB free)|
|Process Argv|.|
|Screen Reader|yes|
|VM|0%|
</details>Extensions: none

cc -isidorn
